ui/src/main/java/com/dat3m/ui/result/ReachabilityResult.java

```diff
@@ -120,15 +120,18 @@ private void buildVerdict(Program p, Result result, ProverEnvironment prover, En
         StringBuilder sb = new StringBuilder();
         Model model = (result == FAIL && !p.getAss().getInvert()) || (result == PASS && p.getAss().getInvert()) ? prover.getModel() : null;
     	for(Axiom ax : wmm.getAxioms()) {
-        	if(ax.isFlagged() && model != null && TRUE.equals(model.evaluate(CAT.getSMTVariable(ax, ctx)))) {
+        	if(ax.isFlagged() && model != null && TRUE.equals(model.evaluate(CAT.getSMTVariable(ax, ctx.getFormulaManager())))) {
```

Comment:
In some PR I changed `Property.getSMTVariable` to take an `EncodingContext` while you changed it to taking a `FormulaManager`. I don't mind either way for now, but this causes a compilation error at the moment.